azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/PartitionPump.java

```diff
@@ -58,7 +61,7 @@ Void startPump()
             	// If the processor won't create or open, only thing we can do here is pass the buck.
             	// Null it out so we don't try to operate on it further.
             	this.processor = null;
-            	this.host.logWithHostAndPartition(Level.SEVERE, this.partitionContext, "Failed " + action, e);
+            	TRACE_LOGGER.warn(LoggingUtils.withHostAndPartition(this.host.getHostName(), this.partitionContext, "Failed " + action), e);
```

Comment:
Suggest error -- application writer will always want to log this. This is catastrophic, the sort of problem that should set off automated alerts if the user has them!